utils/swaggerDefinition.js

```diff
@@ -109,7 +112,7 @@ const swaggerOptions = {
           }
         },
         errors: {
-          unAuthorized: {
+          unauthorized: {
```

Comment:
Make it `unAuthorized`, as it is referenced in `auth.js` and `healthcheck.js` too, the reference might fail there.